Runtime/Core/SelectionPickerSettings.cs

```diff
@@ -74,9 +105,9 @@ public bool Equals(SceneSelection other)
             if (ReferenceEquals(this, other)) return true;
             return Equals(gameObject, other.gameObject)
                 && Equals(mesh, other.mesh)
-                && vertex == other.vertex
-                && edge.Equals(other.edge)
-                && Equals(face, other.face);
+                && vertexes == other.vertexes
+                && edges.Equals(other.edges)
+                && Equals(faces, other.faces);
```

Comment:
This is doing a reference comparison for the lists, which probably isn't what you want here. Think of it like comparing the pointer address vs. the contents of the collections.